www-src/app/replicator/replicator_config.coffee

```diff
@@ -44,18 +45,34 @@ module.exports = class ReplicatorConfig extends Backbone.Model
         else
             return 'https'
 
+    makeDSUrl: (path) ->
+        json: true
+        url: "#{@getScheme()}://#{@get("deviceName")}:#{@get('devicePassword')}" + "@" + @get('cozyURL') + '/ds-api' + path
```

Comment:
@jacquarg it's a bit weird to mix string interpolation and `+`, and to mix single and double quotes on the same line